src/pages/home.tsx

```diff
@@ -87,6 +87,16 @@ const Home = () => {
               &apos;s profile
             </button>
           )}
+
+          <button
+            className="flex items-center bg-slate-700 hover:bg-slate-700/70 hover:text-orange text-white gap-2 p-1.5 px-3 w-full rounded-sm font-medium text-sm"
+            onClick={() => {
+              setCurrentPage("resources");
+            }}
+          >
+            <HiArrowTopRightOnSquare />
```

Comment:
This icon seemed good to me to be used for the resources page but as it is the same as the `Go to Highlights feed` and the `Go to Dashboard` link icons, it may create slight confusion for the users.

Reply:
This icon is used for external links, so it might not be suitable here.

Reply:
Maybe checkout the GrResources icon.

Reply:
Much better

Reply:
d> Maybe checkout the GrResources icon here - https://react-icons.github.io/react-icons/search?q=resou

There seems to be some issue with the grommet icons. I tried using the GrResources icon and changing its color to white. Although it's parent element has a text-white class, it is not getting applied to GrResources.
GrResources accepts a color prop as well but even that does not seem to work.
Using HiSquare3Stack3D from Heroicons 2 for now. Both look almost similar.

HiSquare3Stack3D
![image](https://github.com/open-sauced/ai/assets/36790357/308709d9-ab1b-44ed-a756-e96e5a9d68c0)

GrResources
![image](https://github.com/open-sauced/ai/assets/36790357/4177959f-c2ce-4977-af5e-7fd24c39fb8a)

Reply:
Looking good!